course-microservice/src/main/resources/application.yml

```
@@ -31,4 +31,4 @@ eureka:
     fetch-registry: true
     register-with-eureka: true
     serviceUrl:
-      defaultZone: http://localhost:8070/eureka/
\ No newline at end of file
+      defaultZone: ${EUREKA_SERVER_URL:http://localhost:8070/eureka/}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)